core-services/jira-lifecycle-plugin/config.yaml

```diff
@@ -536,6 +536,18 @@ orgs:
         branches:
           '*':
             skip_target_version_check: true
+      kubernetes-sigs-kueue:
+        branches:
+          '*':
+            validate_by_default: false
+      kueue-operator:
+        branches:
+          '*':
+            validate_by_default: false
+      kueue-fbc:
+        branches:
+          '*':
+            validate_by_default: false
```

Comment:
I would recommend that these get bumped upto line 527 (to fall in line alphabetically).
You'll also need to copy the same block down under the `openshift-priv` organisation as well.